v3.0/reference/system-databases/information-schema.md

```diff
@@ -2,7 +2,6 @@
 title: Information Schema
 summary: Learn how to use Information Schema in TiDB.
 category: reference
-aliases: ['/docs/sql/information-schema/'] 
```

Comment:
Our convention is to keep the aliases in the v3.0 docs (since this is the default version). I assume it may have got lost by copy+pasting from dev?

Reply:
Yes, sorry